lib/rtcpeerconnection/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-if (typeof RTCPeerConnection !== 'undefined') {
```

Comment:
we should all this checks for webRTC related types into isWebRTCSupported and use it here.

Reply:
@makarandp0 Since this module depends on a specific API, its OK to check on only that API.